### ISSUE TITLE
Better feedback when copy board to clipboard

### ISSUE THREAD

**Issue by glauberramos** [funretro/distributed#227]

**Comment by highskillzz** [comment]
is this issue taken ?

**Comment by glauberramos** [comment]
Hey @highskillzz, it was not taken yet. If you want to do it, please go ahead. I was thinking something like a toast message that shows on the top right of the screen, then after sometime disappears.

**Comment by pankti-m** [comment]
I would like to pick up this bug if is not already taken.

**Comment by glauberramos** [comment]
Please go ahead @pankti-m !

**Comment by pankti-m** [comment]
There are multiple libraries for toast messages in Angular - angularjs-toast, toastr, ng-toast are some which I came across while searching.

I also found a couple of tutorials which simply use 'hideDelay'.

I am new to Angular so would like to know if one method is preferred over another.

Is

**Comment by glauberramos** [comment]
I think you can use toastr. It looks like a nice library.

**Comment by pankti-m** [comment]
I have created the above commit with my first draft of the change, but it seems to be missing something.

The site does not load and the I see the following error on the debugger toolbar - [$injector:modulerr]

I am not sure if I have included the 'toastr' dependency correctly at all the required places.  I am new to Angular so unaware of some basics.  Any feedback/guidance on this patch will be helpful.

Thanks
